sql/ddl_internal.sql

```diff
@@ -465,7 +465,36 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.verify_time_column_constraint(
```

Comment:
Should this function be named differently now that it is actually adding the not null constraint rather than just verifying it is there?